src/Workspaces/Remote/ServiceHub/CodeLensOOP/ReferenceEntryFieldNames.cs

```diff
@@ -0,0 +1,23 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+namespace Microsoft.CodeAnalysis.Remote.CodeLensOOP
+{
+    /// <summary>
+    /// this is well known consts in reference codelens in vs part.
+    /// </summary>
+    internal static class ReferenceEntryFieldNames
```

Comment:
@lineralpha shouldn't be provided by vssdk?

Reply:
yeah. the ms.vs.language defines this.